CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ and this project adheres to
 
 - TLS encryption for node-to-node and client-to-node connections.
   ([#31](https://github.com/mesosphere/kudo-cassandra-operator/pull/31))
+- Bump cassandra exporter version and no longer chown it.
```

Comment:
Probably makes sense to move this to the "changed" changelog section, and write it in active voice for consistency? (e.g. "Bumped Cassandra Prometheus exporter to 2.3.2")